test/resources/case1

```diff
@@ -0,0 +1 @@
+Test3
```

Comment:
This is OK, but could you move these files under `test/resources/grep/case1*`? That should fix the other failures.

Reply:
OK, I moved it, and sorry for my thoughtless actions.